Doc/library/dis.rst

```diff
@@ -293,13 +293,17 @@ details of bytecode instructions as :class:`Instruction` instances:
 
    .. data:: argval
 
-      resolved arg value (if known), otherwise same as arg
+      resolved arg value (if known), otherwise ``dis.UNKNOWN``
 
+      .. versionchanged:: 3.11
+         Unknown values are now set to an appropriate sentinel.
 
    .. data:: argrepr
 
-      human readable description of operation argument
+      human readable description of operation argument (if known).
```

Comment:
This should state what it is if not known -- you cannot rely on the "versionchanged" note for that. I wouldn't leave it unset, but I'd be okay if it was one of: `None`, `""`, `"<unknown>"` or `dis.UNKNOWN`. BTW, what is it for opcodes that don't have an oparg (like `RETURN_VALUE`)?

Reply:
I was planning to define what it is, but then I need to assert that in a unit test and it's actually not possible to create an Instruction object with an UNKNOWN argval through the public API (unless I construct it directly, but then what am I actually checking?).  You can generate them by calling get_instruments, but that only works for things that do have co_consts and co_names (you can't give it just the bytes of the code).

Reply:
get_instruments  --> get_instructions (typing instrument is muscle memory you get from working in a bank. I did it several times today)

Reply:
Maybe we don't need to document the "unknown value" case at all here, since it's an internal thing of the dis module?

Reply:
> BTW, what is it for opcodes that don't have an oparg (like `RETURN_VALUE`)?

It's an empty string.